clients/lsp-rust.el

```diff
@@ -68,7 +68,12 @@
   :group 'lsp-rust-rls
   :package-version '(lsp-mode . "6.1"))
 
-(defcustom lsp-rust-library-directories '("~/.cargo/registry/src" "~/.rustup/toolchains")
+(defcustom lsp-rust-library-directories
+  (let ((home-dir (pcase system-type
+                    ('windows-nt (getenv "USERPROFILE"))
```

Comment:
The `~` should expand to `%USERPROFILE%` under windows, why do we need this change? Just to expand the path?

Reply:
You are right. Sorry for the noisy. 😓